AndroidManifest.xml

```diff
@@ -50,6 +50,8 @@
                  android:largeHeap="true"
                  tools:ignore="GoogleAppIndexingWarning"
                  android:requestLegacyExternalStorage="true"
+                 tools:targetApi="TIRAMISU"
```

Comment:
just for interest, what is the ` tools:targetApi="TIRAMISU"` needed for now?

Reply:
to hint the Android Studio linter, because hasFragileUserData was added on android 10 and we have minSDK 16 so the property has no effect in <10